spec.emu

```diff
@@ -56,15 +56,15 @@ contributors: Google, Ecma International
         </tr>
         <tr>
           <td>*"coptic"*</td>
+          <td>*"am"*</td>
           <td>*"coptic"*</td>
-          <td></td>
           <td>*1*<sub>𝔽</sub></td>
           <td>*+&infin;*<sub>𝔽</sub></td>
         </tr>
         <tr>
           <td>*"coptic"*</td>
+          <td>*"am-inverse"*</td>
```

Comment:
Should be `bd` (if we're proceeding assuming the removal of BD is still under discussion in CLDR) or the whole row should be deleted (if we're assuming it will go ahead)

Reply:
I think we can assume that BD is being removed. https://github.com/unicode-org/cldr/pull/4620

Reply:
My understanding was that the popular of AD would also prove to be a challenge for BD.

Reply:
In that case, delete this row and change the allowed minimum of `am` to −∞